build.sbt

```diff
@@ -14,6 +14,11 @@ import com.typesafe.tools.mima.core._
 // Turn off "Resolving" log messages that clutter build logs
 ivyLoggingLevel in ThisBuild := UpdateLogging.Quiet
 
+def defineSbtVersion(scalaBinVer: String): String = scalaBinVer match {
+  case "2.12" => "1.0.2"
+  case _ => "0.13.16"
```

Comment:
Is that correct? When compiling for scala 2.11 we will fallback to sbt 0.13.16?

Reply:
This is for cross-building the plugins. sbt 1.0 requires plugins to be built with 2.12, and sbt 0.13 requires plugins to be built with 2.10 (not 2.11).